### PR TITLE
New Latin Declensions Cheat Sheet

### PR DESCRIPTION
**Pull request by cjmcnamara**
https://duck.co/ia/view/latindecl_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@cjmcnamara Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@cjmcnamara thanks again for creating the Latin Declensions Cheat Sheet! You’ve done a really good job here and we'd like you to help our community tackle the next big challenge - our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements 

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=latindecl cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/latindecl_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@cjmcnamara Thanks! This LGTM :+1:

File: share/goodie/cheat_sheets/json/latindecl.json [hunk 1]
```diff
@@ -0,0 +1,430 @@
+{
+    "id": "latindecl_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/latindecl.json]: @cjmcnamara just so you know, the default trigger for a cheat sheet comes from the filename and the filename is determined by the IA Page ID.

So right now the default trigger for this IA is `latindecl`. If you'd like to support the query `latin noun declensions` you can add it as an alias, or we can change the ID of the cheat sheet in the JSON and on the community platform to match

**Comment by daxtheduck** [comment]
@cjmcnamara this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

File: share/goodie/cheat_sheets/json/latindecl.json (hunk 1 above)
Comment (cjmcnamara) [inline share/goodie/cheat_sheets/json/latindecl.json]: @moollaza Great, thanks! I'll consider adding some useful aliases to catch all the searches.

**Comment by zekiel** [comment]
Hey @cjmcnamara! This went live (just fyi) https://duckduckgo.com/?q=latindecl+cheat+sheet&ia=cheatsheet

**Pull request merged**